README.md

```diff
@@ -81,7 +89,7 @@ This will take all the migrations that haven't been applied and runs them agains
 For development purposes, The PostgreSQL can be connected to via its Docker container shell, to view database contents or metadata. To get a bash shell, run:
 
 ```
-docker exec -it back-end_db_1 /bin/bash
+docker exec -it back-end-db-1 /bin/bash
```

Comment:
On Mac, this command works with the old underscore name, not the updated one. Is the container name different for linux? If so, should we add a comment for both cases

Reply:
Oh yeah it's different on linux, I'll add a comment I just assumed it was the same everywhere

Reply:
Thanks Russ!